rpc/src/rpc_subscriptions.rs

```diff
@@ -719,27 +793,42 @@ impl RpcSubscriptions {
                             }
                         }
                         NotificationEntry::Bank(commitment_slots) => {
+                            const SOURCE: &str = "bank";
                             RpcSubscriptions::notify_accounts_logs_programs_signatures(
```

Comment:
Feel free to rename this method when you include block. It is a mouthful.
Perhaps just `notify_watchers`?

Reply:
good idea